pkg/cmd/resource/namespace.go

```diff
@@ -0,0 +1,88 @@
+package resource
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/stripe/stripe-cli/pkg/ansi"
+)
+
+//
+// Public types
+//
+
+// NamespaceCmd represents namespace commands. Namespace commands are top-level
+// commands that are simply containers for resource commands.
```

Comment:
Would it be possible to specify an example of what a namespace command might be here? I assume it's something like "issuing" or "terminal"?

Reply:
Yes, exactly. Will update the comment 👍